game-localization/localization_tr.txt

```diff
@@ -0,0 +1,39 @@
+// GENERIC
+generic_by : tarafından
```

Comment:
Oh wow, is a lot of characters for "by". 😅
We may have to support multiline labels in buttons for that to fit... 
<img width="252" alt="Screen Shot 2021-11-27 at 23 10 14" src="https://user-images.githubusercontent.com/6775074/143722039-737643c0-c297-44b9-8a4c-5f8d2c08f347.png">